services/resource-coordinator/src/drivers/kubernetes/KubernetesDriver.js

```diff
@@ -1,7 +1,7 @@
 const { BaseDriver } = require('@openintegrationhub/resource-coordinator');
 const uuid = require('uuid/v4');
 const _ = require('lodash');
-const KubernetesRunningApp = require('./KubernetesRunningNode');
+const KubernetesRunningNode = require('./KubernetesRunningNode');
```

Comment:
@anton-kotenko I agree with you. I was trying to get rid of `app`, because in my opinion it's too abstract. I want to name this thing, so that it would be clear, that it represents a flow node running in Kubernetes. I'll come with new name suggestions a bit later.

Reply:
@anton-kotenko how about `RunningFlowNode` for the base class, and `KubernetesRunningFlowNode` for the K8s specific?